core/container.go

```diff
@@ -206,6 +213,9 @@ func (container *Container) From(ctx context.Context, gw bkgw.Client, addr strin
 	_, cfgBytes, err := gw.ResolveImageConfig(ctx, ref, llb.ResolveImageConfigOpt{
 		Platform:    &platform,
 		ResolveMode: llb.ResolveModeDefault.String(),
+		// FIXME: `ResolveImageConfig` doesn't support progress groups. As a workaround, we inject
+		// the group in the vertex name.
+		LogName: fmt.Sprintf("[dagger.group:%s] resolve image config for %s", group.ID(), ref),
```

Comment:
Makes sense, just curious: does something (perhaps your CLI demo) parse this out? Don't see any other reference to `dagger.group` in this PR.

Reply:
On this note, if it's straightforward I think we should update our current palin progress code to strip this out when present, otherwise the output looks like: `#3 [dagger.group:["from alpine:3.16.2"]] resolve image config for docker.io/library/alpine:3.16.2` which is kind of confusing.